packages/stops-overlay/src/StopsOverlay.story.js

```diff
@@ -123,3 +140,17 @@ export default {
 export const Default = () => <Example />;
 
 export const WithCustomMarkers = () => <Example symbols={customSymbols} />;
+
+export const FlexStops = () => (
+  <Example
+    filterStops={false}
+    stops={mockFlexStops}
+    mapCenter={[33.85, -84.61]}
+    symbols={[
+      {
+        minZoom: 5,
+        symbol: ExampleMarker
+      }
+    ]}
```

Comment:
Nit picky but these props could be sorted maybe: 
```suggestion
    filterStops={false}
    mapCenter={[33.85, -84.61]}
    stops={mockFlexStops}
    symbols={[
      {
        minZoom: 5,
        symbol: ExampleMarker
      }
    ]}
```